devito/finite_differences/differentiable.py

```diff
@@ -623,6 +623,24 @@ class Mod(DifferentiableOp, sympy.Mod):
     __sympy_class__ = sympy.Mod
 
 
+class SafeInv(Differentiable, sympy.core.function.Application):
+    _fd_priority = 0
+
+    def __new__(cls, val, base, **kwargs):
+        newobj = super().__new__(cls, val, base)
+        return newobj
+
+    def _evaluate(self, **kwargs):
```

Comment:
why is this override needed? it should automatically pick up

https://github.com/devitocodes/devito/blob/9b8a144d8418102cc48e0e09fd0be2d53a34bcfe/devito/types/lazy.py#L45

Reply:
I honestly don't rember was having issues without it I'll double check, same below